tools/benchmark_tool/openvino/tools/benchmark/utils/inputs_filling.py

```diff
@@ -37,6 +38,15 @@ def get_next_input(self):
     def get_next_batch_size(self):
         return self.batch_sizes[self.current_group_id]
 
+def import_cv2():
+    global cv2
+    
+    try:
+        import cv2
+    except ImportError as ex:
+        raise Exception("Loading images requires opencv-python module. " \
+            "Please install it before continuing or run benchmark without "\
```

Comment:
Do you plan to remove it from requirements?
Without it installation benchmark_app via setup.py or pip still will install opencv in user space

Reply:
Let's not remove it from requirements yet. I like having this exception because even though you will have opencv-python installed after `pip install openvino-dev`, you may at some point remove it (when running into issues with not installed libraries) or you may use benchmark_app from source. It's nice to have a clear error message in that case.

Reply:
but what is the benefits from this PR if we don't remove requirement from benchmark_app? 
It does not resolve the issue on systems without X11 then..

Reply:
opencv doesn't fail on install, only on import. With this PR benchmark_app works normally with the default random methods, and users get the message to install opencv when they specifically use the `-i` option to use images. Currently opencv is imported on load so if you do not have libGL installed, you now get an unclear error even when using benchmark_app with random inputs. This PR is not the perfect solution, but it is a good fix for now that does not have unwanted side effects.

Reply:
@PiotrKrzem You get an ImportError if you do have opencv-python installed but do not have LibGL installed. So this message is incorrect in that case. We should tell users who do have OpenCV, but still get an ImportError, to uninstall opencv-python and install opencv-python-headless instead :-( It is not a great solution. 

```
$ benchmark_app -m ov_model.xml -d CPU -hint latency -t 30
Traceback (most recent call last):
  File "/home/ubuntu/venvs/openvino_env/bin/benchmark_app", line 5, in <module>
    from openvino.tools.benchmark.main import main
  File "/home/ubuntu/venvs/openvino_env/lib/python3.8/site-packages/openvino/tools/benchmark/main.py", line 14, in <module>
    from openvino.tools.benchmark.utils.inputs_filling import get_input_data
  File "/home/ubuntu/venvs/openvino_env/lib/python3.8/site-packages/openvino/tools/benchmark/utils/inputs_filling.py", line 5, in <module>
    import cv2
  File "/home/ubuntu/venvs/openvino_env/lib/python3.8/site-packages/cv2/__init__.py", line 8, in <module>
    from .cv2 import *
ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

Reply:
Done!